### PR TITLE
Moe/fix nano selector

### PR DESCRIPTION
**Pull request by Moejoe90**
image

### CODE REVIEW

File: app/components/walletSelector/WalletLedgerChooserTableAdvanced.spec.tsx [hunk 1]
```diff
@@ -1,38 +1,44 @@
...
 import * as React from "react";
 import { spy } from "sinon";
 
 import { AccountRow, WalletLedgerChooserTableAdvanced } from "./WalletLedgerChooserTableAdvanced";
 
+import BigNumber from "bignumber.js";
 import { tid } from "../../../test/testUtils";
+import { MONEY_DECIMALS } from "../../config/constants";
+
+const ethWeiConvertionConstant = new BigNumber(10).pow(MONEY_DECIMALS);
```
Comment (Moejoe90) [inline app/components/walletSelector/WalletLedgerChooserTableAdvanced.spec.tsx]: I want to put this in the `constants file` but i'm not sure if you want imports there

Comment (krzkaczor) [reply app/components/walletSelector/WalletLedgerChooserTableAdvanced.spec.tsx]: I added it already in the PR that you were supposed to check 😆 it's called Q18

Comment (Moejoe90) [reply app/components/walletSelector/WalletLedgerChooserTableAdvanced.spec.tsx]: :dancer:

File: app/components/walletSelector/WalletLedgerChooserTableAdvanced.spec.tsx [hunk 2]
```diff
@@ -1,38 +1,43 @@
...
-      balanceNEU: "0",
+      balanceETH: weiBalance,
+      balanceNEU: neuWeiBalance,
     },
     {
       address: "0xB2A0e2688c5A82bEEe6818F5a3D206680FdFD75d",
       derivationPath: "44'/60'/0'/1",
       balanceETH: "0",
-      balanceNEU: "0",
+      balanceNEU: "0.0000",
```
Comment (krzkaczor) [inline app/components/walletSelector/WalletLedgerChooserTableAdvanced.spec.tsx]: this should be neu value represented in weis so no dots

File: app/components/walletSelector/WalletLedgerChooserTableSimple.spec.tsx [hunk 3]
```diff
@@ -58,16 +63,25 @@ describe("<WalletLedgerChooserTableSimple />", () => {
...
       const renderedAddress = accountRow.find(tid("account-address"));
       expect(renderedAddress.text()).to.be.eq(account.address);
       const renderedBalanceETH = accountRow.find(tid("account-balance-eth"));
-      expect(renderedBalanceETH.text()).to.be.eq(account.balanceETH + " ETH");
+      expect(renderedBalanceETH.text()).to.be.eq(`${ethBalance} ETH`);
       const renderedBalanceNEU = accountRow.find(tid("account-balance-neu"));
-      expect(renderedBalanceNEU.text()).to.be.eq(account.balanceNEU + " NEU");
+      expect(renderedBalanceNEU.text()).to.be.eq(`${neuBalance} NEU`);
+    });
+    it("should handle click", () => {
```
Comment (krzkaczor) [inline app/components/walletSelector/WalletLedgerChooserTableSimple.spec.tsx]: new line between test cases bitte

File: app/components/walletSelector/WalletLedgerChooserTableAdvanced.spec.tsx (hunk 2 above)
Comment (Moejoe90) [reply app/components/walletSelector/WalletLedgerChooserTableAdvanced.spec.tsx]: Good that you pointed this, for some reason it passed from my mind to make connect this with the whole calculation process

File: app/components/walletSelector/WalletLedgerChooserTableSimple.spec.tsx (hunk 3 above)
Comment (Moejoe90) [reply app/components/walletSelector/WalletLedgerChooserTableSimple.spec.tsx]: Done

**Pull request merged**